kiwi/boot/image/base.py

```diff
@@ -446,7 +446,7 @@ def _get_boot_image_output_file_format_from_dracut_code(self):
     def _get_boot_image_output_file_format_from_existing_file(
         self, kernel_version
     ):
-        for initrd_file in glob.iglob(self.boot_root_directory + '/init*'):
+        for initrd_file in glob.iglob(self.boot_root_directory + '/boot/init*'):
```

Comment:
I am wondering if there is any use case in which this might not be the appropriate path. In the Disk tests I did this was correct.

Reply:
The reason why there is an extra self.boot_root_directory is because of the support for custom kiwi initrds. If you look at kiwi/boot/image/builtin_kiwi.py you see the difference in construction. However I think your change is correct in any way. self.boot_root_directory is always pointing to a linux root (/)